src/tera.rs

```diff
@@ -123,48 +124,51 @@ impl Tera {
             .map(|(n, t)| (n.clone(), t.clone())) // TODO: avoid that clone
             .collect();
 
-        let mut errors = String::new();
-
-        let dir = self.glob.clone().unwrap();
-        // We clean the filename by removing the dir given
-        // to Tera so users don't have to prefix everytime
-        let mut parent_dir = dir.split_at(dir.find('*').unwrap()).0;
-        // Remove `./` from the glob if used as it would cause an error in strip_prefix
-        if parent_dir.starts_with("./") {
-            parent_dir = &parent_dir[2..];
-        }
+        #[cfg(feature = "globs")]
```

Comment:
This is a subtle change in behavior that caught me off guard at first. I thought this was a behavioral breaking change at first, but then i realized that you've made `globs` a default feature, so it's subtly not a breaking change. 👍

Reply:
It is a breaking change for anyone using default-features=false